From 67205c8fca0b4fc1cf1d99775f624f3c41ae3045 Mon Sep 17 00:00:00 2001
From: Adriana Reus <adriana.reus@nxp.com>
Date: Mon, 26 Feb 2018 17:03:53 +0200
Subject: [PATCH 3378/5242] MLK-17626 Use DEFINE_SPINLOCK for imx_ccm_lock

commit  0ff8d2564c12f3e85cd6a3bc66873823c2ed15f0 from
https://source.codeaurora.org/external/imx/linux-imx.git

If spinlock debug is enabled there are BUG_ON asserts in
place for default values on the spinlock_t members.
DEFINE_SPINLOCK declares and initializes the spinlock and
avoids triggering those.

Signed-off-by: Adriana Reus <adriana.reus@nxp.com>
Tested-by: Marius Vlad <marius-cristian.vlad@nxp.com>
Acked-By: Marius Vlad <marius-cristian.vlad@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-imx8.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/imx/clk-imx8.c b/drivers/clk/imx/clk-imx8.c
index 28c9b0d..73c9457 100644
--- a/drivers/clk/imx/clk-imx8.c
+++ b/drivers/clk/imx/clk-imx8.c
@@ -18,7 +18,7 @@
 
 #include "clk-imx8.h"
 
-spinlock_t imx_ccm_lock;
+DEFINE_SPINLOCK(imx_ccm_lock);
 sc_ipc_t ccm_ipc_handle;
 
 int imx8_clk_mu_init(void)
-- 
1.7.9.5

